src/Tribe/Event_Cleaner.php

```diff
@@ -1,5 +1,5 @@
 <?php
-
+// @todo This only handles update option actions? Not clear, perhaps naming/refactor a bit.
```

Comment:
Either rename it now, or remove this todo.

Reply:
Whoops.